components/apimgt/org.wso2.carbon.apimgt.core/src/main/java/org/wso2/carbon/apimgt/core/dao/impl/ApiDAOImpl.java

```diff
@@ -240,6 +240,49 @@ public String getLastUpdatedTimeOfGatewayConfig(String apiId) throws APIMgtDAOEx
     }
 
     /**
+     * Retrieves summary data of all available APIs with life cycle status that matches the status list provided
+     * which has role based visibility
+     *
+     * @param roles    role list of current user
+     * @param statuses status of APIs to be returned
+     * @return API list
+     * @throws APIMgtDAOException if failed to fetch APIs from database
+     */
+    @Override
+    @SuppressFBWarnings("SQL_PREPARED_STATEMENT_GENERATED_FROM_NONCONSTANT_STRING")
```

Comment:
Fixed in : https://github.com/wso2/carbon-apimgt/pull/3894/commits/b64c0402b3d5cd11b3f3841e4ea648a7a0254ec5